attestation-service/src/token/ear_default_policy.rego

```diff
@@ -103,6 +103,22 @@ hardware := 2 if {
 	input.tdx.quote.body.mr_seam in data.reference.mr_seam
 	input.tdx.quote.body.tcb_svn in data.reference.tcb_svn
 	input.tdx.quote.body.mr_td in data.reference.mr_td
+	# Check TCB status
```

Comment:
move the policy updates to a separate commit.

Reply:
moved